src/main/kotlin/com/waicool20/wai2k/script/modules/combat/maps/Map4_3E.kt

```diff
@@ -63,7 +63,7 @@ class Map4_3E(scriptComponent: ScriptComponent) : HomographyMapRunner(scriptComp
         mapRunnerRegions.planningMode.click(); yield()
 
         logger.info("Selecting ${nodes[0]}")
-        nodes[1].findRegion().click()
+        nodes[0].findRegion().click()
```

Comment:
You sure you want to be clicking node 0?

Reply:
Yup should be heliport node.

![image](https://user-images.githubusercontent.com/16036971/98163433-ae69db00-1e97-11eb-98f6-40e18a7127aa.png)

I still need to crop the image so it fits the size and redo coords

Reply:
Might as well do it in another branch and resubmit a new PR then